src/components/grid-aware/VideoHeader/VideoHeader.stories.js

```diff
@@ -28,5 +28,5 @@ DefaultVideoHeader.args = {
     { text: "Donate", internalLink: "/new/donate" },
     { text: "Volunteer", internalLink: "/new/volunteer" },
   ],
-  playButtonLink: "https://www.youtube.com/watch?v=2aLyGwaRufY",
+  playButtonLink: "https://www.youtube.com/watch?v=V1u2l_8C75k",
```

Comment:
This code doesn't exist anymore in `main`. I replaced this link prop with an onClick handler prop in https://github.com/ShelterTechSF/sheltertech.org/pull/127.

Reply:
👍 Fixed with everyone's help!